modules/rpc/yarpc_options.go

```diff
@@ -31,28 +31,28 @@ const (
 	_onewayInterceptorKey = "yarpcOnewayInboundMiddleware"
 )
 
-// WithInboundMiddleware adds custom YARPC inboundMiddlewares to the module
+// WithInboundMiddleware adds custom YARPC inboundMiddleware to the module
 func WithInboundMiddleware(i ...middleware.UnaryInbound) modules.Option {
 	return func(mci *service.ModuleCreateInfo) error {
-		inboundMiddlewares := inboundMiddlewaresFromCreateInfo(*mci)
-		inboundMiddlewares = append(inboundMiddlewares, i...)
-		mci.Items[_interceptorKey] = inboundMiddlewares
+		inboundMiddleware := inboundMiddlewareFromCreateInfo(*mci)
+		inboundMiddleware = append(inboundMiddleware, i...)
+		mci.Items[_interceptorKey] = inboundMiddleware
 
 		return nil
 	}
 }
 
-// WithOnewayInboundMiddleware adds custom YARPC inboundMiddlewares to the module
+// WithOnewayInboundMiddleware adds custom YARPC inboundMiddleware to the module
```

Comment:
nit: previous comment seemed to be inline with variadic arguments. Can we keep it, or not have variadic arguments here? I have no preference on the definition though:)